### ISSUE TITLE
If dictionary is given as folder its not added to the afl-fuzz command

### ISSUE THREAD

**Issue by wizche** [0xricksanchez/AFL_Runner#48]
When the dictionary is specified as a folder like this:
```
# Token dictionary to use
dictionary = "/tmp/mydict/"
```

The resulting command does not contain `-x /tmp/mydict/` and does not even warn about it. 
When specifying a file it works as intended. 
Additional infos: https://github.com/AFLplusplus/AFLplusplus/blob/stable/dictionaries/README.md

**Comment by 0xricksanchez** [comment]
Yeah this was not supported as I was not using this a whole lot. Supporting this should be an easy fix :)
